src/patternfly/components/Page/examples/index.js

```diff
@@ -33,6 +36,9 @@ export default () => {
       >
         {pageComponentFillExample}
       </Example>
+      <Example heading="Page Component Main Section Modifiers" handlebars={PageComponentMainSectionModifiersRaw}>
```

Comment:
Can you add "Padding" so it's "Page Component Main Section Padding Modifiers", since there are other main section modifiers available that aren't in this example?

Reply:
Should we consider doing an example for those other modifiers then? I think its worth it.